tests/dune

```diff
@@ -2,4 +2,5 @@
   (name tests)
   (deps sample.opam)
   (package opam-file-format)
-  (libraries alcotest opam-file-format))
+  (libraries alcotest opam-file-format)
+  (flags (:standard -warn-error -3)))
```

Comment:
I think it's better to do this in `tests/oldpos.ml` so that it's just that one script which has it?